### PR TITLE
added a listener to the back to login button on the registration scre…

### PR DESCRIPTION
**Pull request by nicolePerey**
…en. Transitions from registration to log in. Also added a constructor to the GoToLoginBListener

### CODE REVIEW

File: src/main/java/version1/gameUtil/screens/RegistrationScreen.java [hunk 1]
```diff
@@ -125,7 +127,8 @@ public RegistrationScreen setRegisterButtonText(String registerButtonText) {
     }
 
     public RegistrationScreen setGoToLoginScreenButtonText(String goToLoginScreenButtonText) {
-        this.goToLoginScreenButton = new JButton(goToLoginScreenButtonText);
+        //this.goToLoginScreenButton = new JButton(goToLoginScreenButtonText);
+        this.goToLoginScreenButton= new MazeButton(goToLoginScreenButtonText, new GoToLoginBListener(gameFrame));
 
         // Testing goTo()
         this.goToLoginScreenButton.addActionListener(e -> gameFrame.goTo(new LoginScreen(gameFrame)));
```
Comment (BenMob) [inline src/main/java/version1/gameUtil/screens/RegistrationScreen.java]: Great! Now, this statement can go. 

```// Testing goTo()
        this.goToLoginScreenButton.addActionListener(e -> gameFrame.goTo(new LoginScreen(gameFrame)));```

File: src/main/java/version1/gameUtil/screens/RegistrationScreen.java [hunk 2]
```diff
@@ -125,7 +127,8 @@ public RegistrationScreen setRegisterButtonText(String registerButtonText) {
     }
 
     public RegistrationScreen setGoToLoginScreenButtonText(String goToLoginScreenButtonText) {
-        this.goToLoginScreenButton = new JButton(goToLoginScreenButtonText);
+        //this.goToLoginScreenButton = new JButton(goToLoginScreenButtonText);
```
Comment (BenMob) [inline src/main/java/version1/gameUtil/screens/RegistrationScreen.java]: You can also just remove the commented line all together

**Review by BenMob — CHANGES_REQUESTED** [review]
Just those 2 fixes and this should be good to go. Killing it with continuous delivery haha.

File: src/main/java/version1/gameUtil/screens/RegistrationScreen.java (hunk 1 above)
Comment (nicolePerey) [reply src/main/java/version1/gameUtil/screens/RegistrationScreen.java]: Fixed! Making new pull requests.

File: src/main/java/version1/gameUtil/screens/RegistrationScreen.java (hunk 2 above)
Comment (nicolePerey) [reply src/main/java/version1/gameUtil/screens/RegistrationScreen.java]: Fixed!

**Review by BenMob — APPROVED** [review]

**Pull request merged**